### PR TITLE
Make Rapture repo suffix a build arg with a default value.

### PR DESCRIPTION
**Pull request by qingling128**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -12,7 +14,7 @@ RUN apt-get update \
         ca-certificates \
         adduser \
     # Install google-fluentd at version 1.6.0-1.
```
Comment (igorpeshansky) [inline docker/Dockerfile]: Let's move this comment above the `ARG` (and change `Install` to `Default to`.

File: docker/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,7 @@
 FROM debian:9.5-slim
 
+ARG rapture_repo_suffix=20181011-3
```
Comment (bmoyles0117) [inline docker/Dockerfile]: Can we keep this as "repo_suffix"? I'm not sure anyone understands what rapture means except for us.

Comment (qingling128) [reply docker/Dockerfile]: Done.

File: docker/Dockerfile (hunk 1 above)
Comment (qingling128) [reply docker/Dockerfile]: Done.

File: docker/Dockerfile [hunk 3]
```diff
@@ -1,5 +1,8 @@
 FROM debian:9.5-slim
 
+# Default google-fluentd at version 1.6.0-1.
+ARG repo_suffix=20181011-3
```
Comment (igorpeshansky) [inline docker/Dockerfile]: `ARG` names tend to be uppercase (e.g., [here](https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact)). Should we follow that pattern?

**Pull request merged**